gnomad_qc/v4/annotations/insilico_predictors.py

```diff
@@ -143,7 +144,6 @@ def _load_cadd_raw(cadd_tsv) -> hl.Table:
     logger.info("Number of indels in v3 and not in v4: %s.", indel3.count())
```

Comment:
You removed the other loggers, do you still want this one? Also, why this one, but not the others?

Reply:
I didn't know logging wasn't working, I removed others to make it faster, and this number is the only one I cared about because I put other numbers in the dosstring and I could add them up.

Reply:
Do you still want the logger in there now that it's in the docstring?

Reply:
This one is not, I want it but it's not working and it's already run, I will just remove it.

Reply:
I kept 2 loggers because I kind of need them to monitor the duplicates.